tox.ini

```diff
@@ -3,18 +3,26 @@ skipsdist = true
 envlist = py27, flake8
 
 [testenv]
-passenv = PYTEST_ADDOPTS PYTEST_BASE_URL JENKINS_URL JOB_NAME BUILD_NUMBER
+recreate = True
+passenv = DISPLAY
 deps = -rrequirements/requirements.txt
-commands = pytest tests/test_tracking_protection.py
+commands = pytest --driver=Firefox {posargs}
 
 
 [testenv:flake8]
 skip_install = true
 deps = -rrequirements/flake8.txt
 commands = flake8 {posargs:.}
 
+[flake8]
+exclude = .tox, docs
+
+[testenv:clean]
+whitelist_externals=bash
+commands=
```

Comment:
yes, because afaik, recreate would only "recreate" your .tox caching directory.  All the other cruft would still linger (i.e. *.pyc files, __pycache__ dir, etc.).  clean is a good place to remove anything else you may be leaving behind that you'd like to get rid of between runs.